httpbin/core.py

```diff
@@ -337,7 +337,7 @@ def view_status_code(codes):
     return status_code(code)
 
 
-@app.route('/response-headers')
+@app.route('/response-headers', methods=['GET', 'POST'])
```

Comment:
While this is the simplest change, I realize it is functionally odd because you're submitting a POST to a path with a query string. I may add to this to use both the data and the args.